mailet/standard/src/main/java/org/apache/james/transport/mailets/AddSubjectPrefix.java

```diff
@@ -46,7 +46,7 @@
 
     @Override
     public void init() throws MessagingException {
-        subjectPrefix = (String) getInitParameter(MailetConfiguration.SUBJECT_PREFIX.name());
+        subjectPrefix = getInitParameter("subjectPrefix", String.class);
```

Comment:
Why don't we 'stick' with the previous String version, that is directly the type that we want?